ci/utility/jobs/tag-clive/dev-clive-object-tagging.yml

```diff
@@ -0,0 +1,21 @@
+jobs:
+  - name: clive-object-tagging-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+        passed:
+          - rbac-csv-upload-dev
+      - put: meta
+        resource: meta-development
+      - get: aws-common-infrastructure
+      - get: dataworks-aws-s3-object-tagger
+      - .: (( inject meta.plan.terraform-bootstrap ))
+      - .: (( inject meta.plan.terraform-output-aws-s3-object-tagger ))
+      - .: (( inject meta.plan.terraform-output-common ))
```

Comment:
TF_WORKSPACE to this and the one above please and remove any fallback one in meta